__tests__/app.test.js

```diff
@@ -357,5 +355,27 @@ describe("app",()=>{
             })
         })
     })
-    
+    describe("GET /api/articles?topic=mitch",()=>{
+        test("Status Code: 200 - Should respond with correct status code and articles matching topic query",()=>{
+            return request(app)
+            .get("/api/articles?topic=mitch")
+            .expect(200)
+            .then(({body})=>{
+                const mitchData = body.articles
+                mitchData.forEach((mitch) => {
+                    expect(mitch.topic).toBe("mitch")
+
```

Comment:
I'd consider checking the other properties in this test. You can use a jest matcher such as **toMatchObject** MAke sure to check the length of the array before your forEach 👍